src/resources/triton_python_backend_utils.py

```diff
@@ -340,6 +342,23 @@ def set_max_batch_size(self, max_batch_size):
                 str(max_batch_size))
         else:
             self._model_config["max_batch_size"] = max_batch_size
+            if max_batch_size > 1:
+                self.set_dynamic_batching()
+
+    def set_dynamic_batching(self):
+        """Set dynamic_batching as the scheduler for the model if no scheduler is set.
+           If dynamic_batching is already set, then no action is taken and return success.
```

Comment:
Have you run the formatter? The number of characters should be below 80.

Reply:
Weird. I did, but the formatter didn't correct these lines. Manually done.